wandb/integration/sb3/sb3.py

```diff
@@ -86,6 +96,13 @@ def __init__(
         self.model_save_freq = model_save_freq
         self.model_save_path = model_save_path
         self.gradient_save_freq = gradient_save_freq
+        if gradient_log not in ["gradients", "parameters", "all"]:
+            wandb.termwarn(
+                "`gradient_log` must be one of 'gradients', 'parameters', or 'all', "
+                "falling back to 'all'"
+            )
+            gradient_log = "all"
```

Comment:
our default is `gradient` in watch:
https://github.com/wandb/client/blob/master/wandb/sdk/wandb_watch.py#L19

Reply:
> our default is gradients in watch

I kept the default as it was currently in the SB3 callback (but I agree with your point)

---

> Also the name is confusing

Again, I tried to stay consistent with other arguments (`gradient_save_freq`). Similarly, `gradient_save_freq` is confusing, the name should be something like `log_freq`.

But I agree with you ^^  
So I think another PR is needed where all the arguments are renamed to be more consistent.